### PR TITLE
Fix a typo in dry-initializer/custom-plugins article.

### PR DESCRIPTION
**Pull request by alsemyonov**

### CODE REVIEW

File: source/gems/dry-initializer/custom-plugins.html.md [hunk 1]
```diff
@@ -91,17 +91,19 @@ require 'dry-initializer'
 
 module Dry::Initializer::Coercion
   class Builder
```
Comment (smaximov) [inline source/gems/dry-initializer/custom-plugins.html.md]: I believe `Builder` should inherit from `Dry::Initializer::Plugins::Base`, as the other parts of the docs say.

**Comment by timriley** [comment]
@nepalez Is this still valid?

**Comment by nepalez** [comment]
Nope. Not in `v1.0.0`.
I've simplified the code and removed all plugins because they were a sort of overkill

**Comment by timriley** [comment]
Thanks @nepalez, and thanks @alsemyonov for submitting this one. Sorry it didn't make it through!

**Comment by nepalez** [comment]
@smaximov @timriley 

I've implemented new way of enhancing `dry-initializer`.
You can find a simple example here https://github.com/dry-rb/dry-initializer/releases/tag/v1.1.0
A bit more useful lives at https://github.com/nepalez/dry-initializer-rails

**Pull request closed without merging**